src/components/marketoForms/MarketoForm.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from "react";
+import { MarketoFormStyleOverwrite, SuccessText } from "./styles";
+
+declare global {
+  interface Window {
+    MktoForms2: {
+      loadForm?: (
+        url: string,
+        munchkinId: string,
+        formid: number,
+        onSubmit?: (form: any) => void
+      ) => void;
+    };
+  }
+}
+
+function appendScript(setScriptLoaded) {
+  if (window.MktoForms2) return setScriptLoaded(true);
+
+  const script = document.createElement("script");
+  script.src = "//info.saasquatch.com/js/forms2/js/forms2.min.js";
+  script.onload = () => (window.MktoForms2 ? setScriptLoaded(true) : null);
+  document.body.appendChild(script);
+}
+
+const useMarketo = () => {
```

Comment:
I understand we only have a single Marketo form in the docs right now, but I could potentially see us having more in future. Not saying this has to be done now, but maybe:
- Rename to `useMarketoForm`
- Generalize by taking the form identifiers as props, instead of hard coding

Reply:
Very good point! I am going to update that quickly